Sources/SystemModule/Adapters/Permission+Adapters.swift

```diff
@@ -28,7 +28,7 @@ extension System.Permission.Model: CreateAdapter, UpdateAdapter, PatchAdapter {
     }
 }
 
-extension System.Permission.Model.ColumnNames: ColumnNamesInterface {
+extension System.Permission.Model.ColumnNames: ColumnNamesAdapter {
```

Comment:
Shouldn't this be List(Query)SortKeyAdapter? What do you think @viaszkadi ?

Reply:
thats better